src/main/scala/no/ndla/learningpathapi/controller/LearningpathControllerV2.scala

```diff
@@ -811,5 +811,32 @@ trait LearningpathControllerV2 {
     ) {
       readService.contributors
     }
+
+    post(
+      "/:learningpath_id/update-taxonomy/",
+      operation(
+        apiOperation[LearningPathV2]("updateLearningPathTaxonomy")
```

Comment:
Ser ikkje heilt for meg at vi treng resultatet av dette kallet, men det er bedre å returnere det om vi skulle sjå at vi treng det seinare.

Reply:
Enig, men tenkte det var like lett å returnere det som ikke, så tenker det kan stå :+1: